### ISSUE TITLE
Voice call not working using Laravel

### ISSUE THREAD

**Issue by mwshubham** [twilio/voice-quickstart-server-php#5]
I have created functions for calling like `generateAccessToken`, `makeCall`, `placeCall`, `incomingCall` in my controllers. And I have also created public URL using ngrok and have registered on TwiML.
But when I make a call from android twillio SDK ( `implementation 'com.twilio:voice-android:2.0.4'`) only makeCall is hit and no other functions are called which lead to immediate disconnection of the active call on the android side.
![screenshot](https://user-images.githubusercontent.com/29930308/37972414-ee66ab5c-31f5-11e8-9d86-5eed51e489ee.png)
![screenshot1](https://user-images.githubusercontent.com/29930308/37972415-eebcd950-31f5-11e8-931d-c5ff240dab8b.png)

**Comment by kbagchiGWC** [comment]
@mwshubham 
Are you making a `client to client` call or `client to PSTN` call? If you are making a `client to client` call, is the callee registered to receive incoming call?

[README](https://github.com/twilio/voice-quickstart-android/blob/master/README.md) includes step by step instructions  for [client-to-client-call](https://github.com/twilio/voice-quickstart-android#10-make-client-to-client-call) and [client-to-pstn-call](https://github.com/twilio/voice-quickstart-android#11-make-client-to-pstn-call). Please follow the steps and let me know if this is still an issue.  If it is, please include the CallSid and logs (from the android  app and the server) to debug this further.

**Comment by mwshubham** [comment]
Thanks for your reply. I am actually making a `client to client call` via the user `identity`. And Everything works perfect like token registration and making a voice call on identity and push notification on another device on a fresh LAMP setup with public PHP files as same as SDK.

But the problem is when I create controllers and create public URL via ngrok in that case I am not able to complete a call as the only `makeCall` is triggered no other internal function like `placeCall`, `incomingCall` are called. It would be very helpful if you guide me to set up. I have already added screenshot earlier about controller I am creating.

**Comment by bobiechen-twilio** [comment]
Hi @mwshubham 

Thanks for providing updates.
When you said everything works perfect, was your mobile app hitting a public accessible URL so that you can fetch Access Tokens and the TwiML app can fetch TwiML responses from your server?

The `makeCall` endpoint is the ultimate demonstration of how to exercise the most basic function of the SDK - client to client/number call. It doesn't require the `placeCall` and `incomingCall` endpoints, which are the endpoints we use in the README to show developers how to achieve basic incoming functionality.

For basic functions, you will only need the `accessToken` and `makeCall` endpoints.

For not being able to complete the call via the `makeCall` endpoint:
- Is the TwiML application URL pointing to the `makeCall` endpoint?
- Do you see any error in the [Programmable Voice Dashboard](https://www.twilio.com/console/voice/dashboard)?

-bobie

**Comment by DLNlab** [comment]
Hello, Good Morning


Configure step by step as established in the readme.

But when I enter the url where I have my makeCall.php file, it shows me an HTTP 500 error

Is this normal?

Thank you

**Comment by bobiechen-twilio** [comment]
Hi @DLNlab 

Are you able to see any error message in the console? The `500` code typically means the server endpoint has encountered some internal error, usually code exception.

**Comment by DLNlab** [comment]
I do not have any error that reports me. Unless you print from the vendor bookstore, is this so?

If so, where can I check this?

The funny thing is that only the files in makeCall.php and incoming.php mark me the errors of HTTP ERROR 500

PlaceCall.php works correctly
index.php works correctly
accessToken.php works correctly
config.php works correctly

**Comment by bobiechen-twilio** [comment]
Hi @DLNlab 

Both `makeCall.php` and `incoming.php` work for me. Did you add any modification to the endpoints?

**Comment by DLNlab** [comment]
No, just modify the requested data such as KEY_SECRET, API_KEY, etc.

**Comment by bobiechen-twilio** [comment]
Where are you seeing the `500` errors?

**Comment by rasta321** [comment]
i am having the same problem as well, did you end up figuring it out?
